packages/web-components/src/components/dotcom-shell/__stories__/dotcom-shell.stories.ts

```diff
@@ -0,0 +1,142 @@
+            for the entire design system and ties the products in IBM’s portfolio together in a cohesive and elegant way. The
+            shell is the home of the topmost navigation, where users can quickly and dependably gain their bearings and move
+            between pages.
+            <br />
+            <br />
+            The shell was designed with maximum flexibility built in, to serve the needs of a broad range of products and users.
+            Adopting the shell ensures compliance with IBM design standards, simplifies development efforts, and provides great
+            user experiences. All IBM products built with Carbon are required to use the shell’s header.
+            <br />
+            <br />
+            To better understand the purpose and function of the UI shell, consider the “shell” of MacOS, which contains the Apple
+            menu, top-level navigation, and universal, OS-level controls at the top of the screen, as well as a universal dock
+            along the bottom or side of the screen. The Carbon UI shell is roughly analogous in function to these parts of the Mac
+            UI. For example, the app switcher portion of the shell can be compared to the dock in MacOS.
+          </p>
+          <h2>
+            Header responsive behavior
+          </h2>
+          <p>
+            As a header scales down to fit smaller screen sizes, headers with persistent side nav menus should have the side nav
+            collapse into “hamburger” menu. See the example to better understand responsive behavior of the header.
+          </p>
+          <h2>
+            Secondary navigation
+          </h2>
+          <p>
+            The side-nav contains secondary navigation and fits below the header. It can be configured to be either fixed-width or
+            flexible, with only one level of nested items allowed. Both links and category lists can be used in the side-nav and
+            may be mixed together. There are several configurations of the side-nav, but only one configuration should be used per
+            product section. If tabs are needed on a page when using a side-nav, then the tabs are secondary in hierarchy to the
+            side-nav.
+          </p>
+        </div>
+      </div>
+    </div>
+  </main>
+`;
+
+export const Default = ({ parameters }) => {
+  const { brandName, userStatus, navLinks } = parameters?.props?.['dds-masthead-container'] ?? {};
+  const { langDisplay, language, size: footerSize, legalLinks, links: footerLinks, localeList } =
+    parameters?.props?.['dds-footer-composite'] ?? {};
+  return html`
+    <style>
+      ${mastheadStyles}
+      ${footerStyles}
+    </style>
+    <dds-dotcom-shell-container
+      brand-name="${ifNonNull(brandName)}"
+      language="${ifNonNull(language)}"
+      lang-display="${ifNonNull(langDisplay)}"
+      footer-size="${ifNonNull(footerSize)}"
+      user-status="${ifNonNull(userStatus)}"
```

Comment:
Same for user status, this should be from the ProfileAPI...

Reply:
Yes good point, this comes from profile API, this is exposed for flexibility/testing purpose.

Reply:
I wonder if these are for testing purposes, we should make them private? Adding in the documentation might confuse adopters.

Reply:
I see - Updated: https://github.com/carbon-design-system/ibm-dotcom-library/pull/3505/commits/a5f588240d67d327c90e80a21a43e3122d2814ef